app/public.js

```diff
@@ -70,13 +72,50 @@
         }
       }
     });
+    setEventListeners();
   }
 
   /**
+   * Refresh overlay
+   */
+  function refreshImagesInfo() {
+    hideImagesInfo();
+    showImagesInfo(null);
+  }
+
+  /**
+   * Set event listeners
+   */
+  function setEventListeners() {
+    let width = window.innerWidth;
+    let height = window.innerHeight;
+    let path = window.location.pathname;
+    let update;
+
+    interval = setInterval(function() {
```

Comment:
@simonhearne DoD - should clear interval before creating a new one as we're always create another interval every time showImagesInfo is called